src/content/cloud/volume-snapshots.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Volume Snapshots
+description: Use volume snapshots in your development environments
+sidebar_label: Volume Snapshots
+id: use-volume-snapshots
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Volume snapshots allows you to initialize persistent volume claims with the contents of a preexisting volume snapshot.
+Use volume snapshots when working with large datasets or if you want to create development environments with real data coming from your production or staging environments.
+
+## Requirements
+
+<Tabs
+  defaultValue="self-hosted"
+  values={[
+    { label: 'Self-Hosted', value: 'self-hosted', },
+    { label: 'SaaS', value: 'saas', },
+  ]}
+>
+<TabItem value="self-hosted">
+
+Follow [this guide](/docs/self-hosted/administration/volume-snapshots/) to enable Volume Snapshots in your cluster.
+
+Use a storage class provisioned by a CSI driver that [supports volume snapshots](https://kubernetes-csi.github.io/docs/snapshot-restore-feature.html) for the source persistent volumes of your volume snapshots.
+
+</TabItem>
+
+<TabItem value="saas">
+If your instance is hosted by Okteto, Volume Snapshots are enabled by default.
+
+Use one of the following storage class for the source persistent volumes of your volume snapshots:
+
+- `csi-okteto-standard`: backed by standard hard disk drives (HDD)
+- `csi-okteto-ssd`: backed by solid-state drives (SDD)
+
+</TabItem>
+</Tabs>
+
+## Using Volume Snapshots in your Development Environment
+
+Okteto enables developers to initialize persistent volume claims with the contents of a pre-existing volume snapshot.
+The volume snapshot is created from a persistent volume claim and it can contain database backups, big files, images, a copy of your staging data, etc.
+
+The steps to clone the data of an existing persistent volume into your develepment environment are:
+
+- [Create the source persistent volume with the right storage class](/docs/cloud/use-volume-snapshots/#creating-the-persistent-volume)
+- [Create a volume snapshot of the source persistent volume](/docs/cloud/use-volume-snapshots/#creating-a-volume-snapshot)
+- [Consume the volume snapshot in a new persistent volume](/docs/cloud/use-volume-snapshots/#consuming-a-volume-snapshot)
+
+### Creating the Persistent Volume
+
```

Comment:
```suggestion
First, create the source persistent volume. This is the data that you want to be able to clone into your development environment.  In the example below we use a database, but this could be anything that uses a volume for storage: databases, ML models, images, etc...
```